### ISSUE TITLE
Ensure comments and understanding are correct

### ISSUE THREAD

**Issue by ras44** [ras44/gcp_jupyterlab_001#1]
I added a few comments to the below code, specifically identifying that what is labeled as TPR is the FWER calculation:

https://github.com/ras44/gcp_jupyterlab_001/blob/acd216caaa6da2d865c05b874943630abc3c1f7d/mac_run.py#L46-L52

**Comment by michaelaclayton** [comment]
yeah... the Zrep just represents predefined values that I want to calculate at.  But I don't really know which I need yet since they represent single test levels...  so I want a function to interpolate on.
  
Zrep/TF -- I define the multivariate TNR as the lower quadrant multivariate CDF -- the probability that all individual tests will be less than the maximum.

If I ran a test, and got minimum single test p-value: p_m, then find N(p_m) in Zrep, TF is the cdf of this and 1-the resulting TF value represents the p-value.

TPR -- df that contains results assuming that all individual tests are shifted the same way
TPROne -- as above, but with only on test affected.

TPR[(corrs[ccorr],dim)] is the multivariate pvalue for the set of tests.
TPRone[(corrs[ccorr],dim)] is the same thing, just put into the other df for easy reference...

**Comment by michaelaclayton** [comment]
yes, my code/way of describing this is still quite convoluted...
